extensions/2.0/Khronos/KHR_displaymapping_pq/README.md

```diff
@@ -0,0 +1,439 @@
+
+Light contribution values above 10000 cd / m2 is discouraged and will be clamped by implementations if not adjusted by scene aperture.  
+
+## Display type
+
+This section describes how to decide if display type is HDR or SDR.  
+
+### HDR capable display
+
+
+If the framebuffer format and colorspace is known to the implementation then a format and colorspace shall be chosen to preserve the range and precision of the SMPTE ST 2084 transfer function.  
+If available, a framebuffer colorspace that is compatible with the color primaries of ITU BT.2020 shall be used.  
+If this colorspace is used the source color images must be converted from linear ITU BT.709 values to ITU BT.2020 linear values, where applicable depending on texture target and source format.  
+
+For HDR output,  a range extension value of 59.5208 and gamma of 2.4 shall be used.  
+
+**Implementation notes**
+
+Conversion of color primaries to ITU BT.2020 could be done after loading of a PNG/JPEG and after the image has been gamma expanded from sRGB to linear.  
+
+
+### SDR capable display
+
+If the framebuffer format or colorspace is not known, or none is available that preserves range, precision and color gamut then lower range framebuffer with sRGB colorspace may be used.  
+This is to allow for compatibility with displays that does not support higher range and/or compatible colorspaces.  
+It also allows for implementations where the details of the framebuffer is not known or available.  
+
+A SDR capable display is defined as having less than 10 bits per pixel for each colorchannel.  
+
+For SDR output,  a range extension value of 46.42 and gamma of 2.4 shall be used.  
+
+## Color source images
+
+If display type is considered to be HDR then color source images shall be converted to BT.2020 as needed.  
+
+
+### Color conversion matrix
+
+Color conversion from BT.709 to BT.2020 is specified in ITU BT.2087  
+https://www.itu.int/rec/R-REC-BT.2087/en
+
+The M2 linear color conversion matrix is defined as:  
+0.6274 0.3293 0.0433  
+0.0691 0.9195 0.0114  
+0.0164 0.0880 0.8956  
+
+
+## Scene aperture
+
+### Parameters
+
+The following parameters are added by the `KHR_displaymapping_pq` extension, it is added with the extension on a scene object:  
+
+| Name              | Type     | Description                                                                                                                                | Required |
+| ----------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| **sceneAperture** | `object` | Scene light adjustment setting, if no value supplied the default value of OFF will be used and scene light values will be kept unmodified. | No       |
```

Comment:
Since the extension object containing this property is defined on a `scene` object, the property should be called `aperture`.

Reply:
Thanks - updated